pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -1515,19 +1515,21 @@ public void removeBacklogQuota(@Suspended final AsyncResponse asyncResponse,
     public Integer getMessageTTL(@PathParam("tenant") String tenant,
                              @PathParam("namespace") String namespace,
                              @PathParam("topic") @Encoded String encodedTopic,
-                             @QueryParam("applied") boolean applied) {
+                             @DefaultValue("false")@QueryParam("applied") boolean applied) {
```

Comment:
The primitive type `boolean`'s default value is false.

Reply:
maybe you want change like this:
@DefaultValue("false") @QueryParam("applied") Boolean applied